collect_app/src/main/java/org/odk/collect/android/preferences/utilities/PreferencesUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.odk.collect.android.preferences.utilities;
+
+import androidx.preference.CheckBoxPreference;
+import androidx.preference.Preference;
+
+public class PreferencesUtils {
+
+    private PreferencesUtils() {
+
+    }
+
+    public static void displayDisabled(CheckBoxPreference preference, boolean displayValue) {
+        preference.setPersistent(false);
```

Comment:
This is clunky but I couldn't find any other way to just display a value in a `CheckBoxPreference`. You'll see the `String` version is much simpler.